chpl/chpl-service/src/main/resources/errors.properties

```diff
@@ -139,6 +139,7 @@ listing.criteria.badParticipantProfessionalExperienceMonths=Professional Experie
 listing.criteria.badParticipantProductExperienceMonths=Product Experience Months is required for participant %s.
 listing.criteria.badParticipantComputerExperienceMonths=Computer Experience Months is required for participant %s.
 listing.criteria.fuzzyMatch=Changed %s for criteria %s from %s to %s.
+listing.criteria.testFunctionalityMisMatch=In Criteria %s, Test Functionality %s is not valid for Practice Type %s.
```

Comment:
There was a request to add "...something is for something settings only and is not valid...", right?